src/main.tsx

```diff
@@ -1,26 +1,44 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
-
+import { MoviePosterPage } from "./components/movie-poster-page";
 import "./index.css";
-import Member from "./components/Member";
-import { NavBar } from "./components/NavBar";
+
+type movie = {
+	title: string;
+	year: number; // Does it make more sense to have the "year" from type "string"?!
+	image: string;
+};
+
+const movieArr: movie[] = [
+	{
+		title: "Wednesday",
+		year: 2024,
+		image: "./public/img/img01.jpg",
```

Comment:
Files in the public directory are served at the root path.
Instead of /public/img/img01.jpg, use /img/img01.jpg.
Change this for all.